lib/ipmi_backend_utils.pm

```diff
@@ -145,8 +146,27 @@ sub umount_installation_disk {
     assert_script_run("ls $mount_point");
 }
 
+#Get the partition where the new installed system is installed to
 sub get_installation_partition {
-    my $partition = script_output("fdisk -l | grep \"^\/dev\/sda\.\*\\\*\" | cut -d ' ' -f 1");
+    my $partition = '';
+
+    #Do not use script_output because when the command fail, script_output dies
+    type_string("fdisk -l | grep \"^\/dev\/sda\.\*\\\*\" | cut -d ' ' -f 1 | tee /dev/$serialdev\n");
+    $partition = wait_serial;
+    save_screenshot;
+    if (is_storage_ng && ($partition eq '')) {
+        record_soft_failure "Partitioner does not mark boot flag, see bsc#1080729!";
```

Comment:
Please, use the bug reference.

```perl
record_soft_failure "bsc#1080729 Partitioner does not mark boot flag!";
```

Reply:
@SergioAtSUSE  Pardon? What do you mean by bug reference? Isn't bug number enough,bsc#1080729 ?

Reply:
oh, sorry, when I was commenting, the code hadn't the reference. It is ok ;)